code/game/machinery/guestpass.dm

```diff
@@ -33,6 +33,7 @@
 
 /obj/machinery/computer/guestpass
 	name = "guest pass terminal"
+	desc = "This console allows staff to give out temporary access to their coworkers."
```

Comment:
```suggestion
	desc = "This console allows staff to give out temporary access to the clown."
```

Reply:
Going to keep the old one for this, i think its needed to portray what it does